packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -2,32 +2,23 @@
 
 import _ from 'lodash'
 import Bluebird from 'bluebird'
-import type playwright from 'playwright-webkit'
 import type { Protocol } from 'devtools-protocol'
 import type ProtocolMapping from 'devtools-protocol/types/protocol-mapping'
 import { cors, uri } from '@packages/network'
 import debugModule from 'debug'
 import { URL } from 'url'
 
-import type { Automation } from '../automation'
 import type { ResourceType, BrowserPreRequest, BrowserResponseReceived } from '@packages/proxy'
 import type { WriteVideoFrame } from '@packages/types'
+import type { Automation } from '../automation'
+import { cookieMatches, CyCookie, CyCookieFilter } from '../automation/util'
```

Comment:
nit: move method import above type imports

Reply:
Not sure we have any precedent for sorting based on type vs code import, but most files have npm/packages import before relative imports, so I'd prefer to stick with that for consistency.

Reply:
oh we def do in files I have touched 😂